src/lib/components/explorer/export/ExportStepper.svelte

```diff
@@ -116,8 +116,11 @@
 
   async function createNamedDataset() {
     try {
-      const datasetName = encodeURIComponent(datasetNameInput);
```

Comment:
We were double-encoding spaces which was causing chaos

Reply:
where is the second time?